src/Components/MovieCard/MovieCard.js

```diff
@@ -1,10 +1,9 @@
 import './MovieCard.css'
 import { Link } from 'react-router-dom'
 
-const MovieCard = ({movieInfo, handleClick}) => {
+const MovieCard = ({movieInfo}) => {
```

Comment:
When I see a lot of "movieInfo.something", it can indicate an opportunity to use destructuring to avoid that repetitive need for the dot notation syntax.